assets/js/blocks/featured-category/style.scss

```diff
@@ -1,5 +1,5 @@
 .wp-block-woocommerce-featured-category {
-	background-color: #000;
+	background-color: $gray-900;
```

Comment:
Just to confirm, this change is unrelated to this fix, right? You are setting the background color to [how it was](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5542/files#diff-b33081124123f537259e21c9344567df96ba2ecf59e2cbc6ff4da1ec61aeb9e1L3) before #5542, correct?

LGTM but just wanted to make sure I'm understanding it. :smile:

Reply:
Yes, I noticed the difference and I fixed it! 👍